package.json

```diff
@@ -9,7 +9,7 @@
   "scripts": {
     "test": "npm run db-init && nyc mocha test/**/**/**.test.js --require babel-core/register --require babel-polyfill --exit",
     "start": "npm run clean && npm run build && node dist/app.js",
-    "start:dev": "nodemon --exec babel-node app.js",
+    "start:dev": "npm run db-init && nodemon --exec babel-node app.js || true",
```

Comment:
@Darthrighteous This seems different from what the rest of the team is using, right?

Reply:
@mbilesanmi Oh yeah. I thought we would always want both scripts running together but upon further discussion with the team, I now see there are cases we might not want to run both. I'll revert it.